resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -975,4 +1005,95 @@ public class ApiClient implements AutoCloseable {
         @Override
         public void close() throws Exception { }
     }
+
+    public static class RetryConfiguration {
+        public long backoffInterval = 300000L;
+        public long defaultDelay = 3000L;
+        public int maxRetryTime = 0;
+        public int maxRetriesBeforeBackoff = 5;
```

Comment:
@PrinceNebulon is there any way you can think of to protect these configuration options from being set outside of the tests? I don't see them in this review, but I was thinking we could make these static final constants but add some getters that the tests could mock

Reply:
Ah, they just didn't load for me. I see them now

Reply:
I think leaving them fully configurable is ok; however, we're only explicitly documenting the max retry time. Because the SDKs are open source and the retry behavior is optional (as is using the SDK to begin with), blocking consumers from configuring certain options will only serve to cause some users to use a custom build of the SDK or avoid using the logic we offer in favor of implementing their own to work around ours. Less static configurations mean less use cases that prevent people from using the SDK.